contracts/Migrator.sol

```diff
@@ -127,20 +127,21 @@ contract Migrator is Governable {
         uint256 newStakeAmount,
         uint256 newStakeDuration
     ) external isSolvent {
-        if (lockupIds.length == 0) {
-            revert LockupIdsRequired();
-        }
-
-        // Unstake
-        (uint256 ogvAmountUnlocked, uint256 rewardsCollected) = ogvStaking.unstakeFrom(msg.sender, lockupIds);
-
-        if (migrateRewards) {
-            // Include rewards if needed
+        if (lockupIds.length > 0) {
+            // Unstake if there are any lockups
+            (uint256 ogvAmountUnlocked, uint256 rewardsCollected) = ogvStaking.unstakeFrom(msg.sender, lockupIds);
+
+            if (migrateRewards) {
+                // Include rewards if needed
+                ogvAmountFromWallet += rewardsCollected;
+            }
+
+            ogvAmountFromWallet += ogvAmountUnlocked;
```

Comment:
This would be clearer if the `ogvAmountFromWallet += ogvAmountUnlocked;` is moved to be right under the `unstakeFrom`, since that's where the `ogvAmountUnlocked` value comes from. I like having variables that have a short lifetime in the code.

Reply:
Done